docs/e2e-tests.md

```diff
@@ -59,8 +29,38 @@ The [E2E test folder](../e2e-tests/web/test) contains two important subdirectori
 When writing new tests create helper classes for each Page you are using in the tests to keep the tests uniform and readable. This is also called the [page object pattern](https://webdriver.io/docs/pageobjects/).
 The tests itself are then created in the test folder and have to end on `*.e2e.ts`.
 
+### Web
+
+### Local Testing
+
+To run the web E2E-tests you first have to start the web app, and then execute the e2e tests:
+
+```
+yarn workspace e2e prepare:web:start
+yarn workspace e2e test:web
+```
+
+This will launch an automated chrome session where you can follow the test execution. If your don't have chrome installed you can adjust the `browserName` (firefox, edge, safari) in the `wdio.conf.ts` in the E2E-test web subdirectory.
```

Comment:
You could link the wdio conf here

Reply:
Could you mention that chromium does not work/only work after changing the browser name?